examples/stacks/spring/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/db_example
+spring.datasource.username=springuser
+spring.datasource.password=password
```

Comment:
nit: put a comment here to remind people to update this username/password and also exclude it from git tracking for production usecases.